src/App.js

```diff
@@ -1,10 +1,13 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState}  from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
-  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.
+  //  You'll need one for the home score and another for the away score.
+  const [h_value, setValue_h] = React.useState(0);
```

Comment:
Ana I love the font you used on this project a lot. Really well done.
Just a side note here you do not need React as it's has already been imported at the top. So you can remove the React. from React.useState.
Otherwise brilliant work.